05-history.md

```diff
@@ -202,6 +202,25 @@ on the other hand,
 moving backward and forward in time becomes much easier.
 
 
+
+> ## Checking out changed files {.challenge}
+>
+> Assume you have started to work on something, like above and git says to discard thing.
+> How can the command above "git checkout -- mars.txt" be described in terms of git?
+>
+> a) take the last snapshot and overwrite my pending changes
+> b) in the history of the repository, go back one step
+> c) clear the staging area from the changes I did to mars.txt
+> d) delete everything I have done since I last committed
+>
```

Comment:
The comments in parentheses are just the reasoning why you would keep an answer, aren't they?

After starting this comment, I realized it might make sense to re-work the question as a whole. I will push a new PR soon and also consider your suggestions.